src/databricks/labs/remorph/snow/sql_transpiler.py

```diff
@@ -22,13 +23,13 @@ def transpile(
 
         return TranspilationResult(transpiled_sql, error_list)
 
-    def parse(self, sql: str, file_name: str) -> (list[Expression] | None, ParserError | None):
+    def parse(self, sql: str, file_name: str) -> tuple[list[Expression | None] | None, ParserError | None]:
```

Comment:
```suggestion
    def parse(self, sql: str, file_name: str) -> tuple[list[Expression] | None, ParserError | None]:
```

Reply:
`sqlglot` `parse` returns `list[Expression | None]`, so couldn't change this.

```
 returns parse(
    sql: str, read: DialectType = None, dialect: DialectType = None, **opts
) -> t.List[t.Optional[Expression]]
```